babel-defines.js

```diff
@@ -1,13 +0,0 @@
-const shared = {
-  __DEV__: "process.env.NODE_ENV !== 'production'"
```

Comment:
TODO: If this is still needed, provide it via [@rollup/plugin-replace](https://www.npmjs.com/package/@rollup/plugin-replace) (for `yarn run dist:rollup` builds) and via `sed` or similar (for `yarn run dist:transpile:*` builds).